transformers/test_data/vat_slip.go

```diff
@@ -95,19 +93,17 @@ var VatSlipHeaderSyncLogWithNegativeWad = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var VatSlipModelWithNegativeWad = shared.InsertionModel{
+var VatSlipModelWithNegativeWad = event.InsertionModel{
 	SchemaName: constants.MakerSchema,
 	TableName:  constants.VatSlipTable,
-	OrderedColumns: []string{
-		constants.HeaderFK, string(constants.IlkFK), "usr", "wad", constants.LogFK,
-	},
-	ColumnValues: shared.ColumnValues{
-		"usr":              "0xFc7440E2Ed4A3AEb14d40c00f02a14221Be0474d",
-		"wad":              "-5000000000000000",
-		constants.HeaderFK: VatSlipHeaderSyncLogWithNegativeWad.HeaderID,
-		constants.LogFK:    VatSlipHeaderSyncLogWithNegativeWad.ID,
+	OrderedColumns: []event.ColumnName{
+		constants.HeaderFK, constants.IlkColumn, constants.UsrColumn, constants.WadColumn, constants.LogFK,
 	},
-	ForeignKeyValues: shared.ForeignKeyValues{
-		constants.IlkFK: "0x4554482d41000000000000000000000000000000000000000000000000000000",
+	ColumnValues: event.ColumnValues{
+		constants.UsrColumn: "0xFc7440E2Ed4A3AEb14d40c00f02a14221Be0474d",
+		constants.WadColumn: "-5000000000000000",
+		constants.HeaderFK:  VatSlipHeaderSyncLogWithNegativeWad.HeaderID,
+		constants.LogFK:     VatSlipHeaderSyncLogWithNegativeWad.ID,
```

Comment:
I think we probably want to use `events.HeaderFK` and `events.LogFK` here and up on lines 61, 66, 67 and 100.